### PR TITLE
disable materials

### PR DESCRIPTION
**Pull request by mikesndrs**
**Summary of changes**
Disable scraped materials if they are not relevant

**Checklist**

- [x] I have read and followed the [CONTRIBUTING](https://github.com/ElixirTeSS/TeSS/blob/master/CONTRIBUTING.md) guide.
- [x] I confirm that I have the authority necessary to make this contribution on behalf of its copyright owner and agree
  to license it to the TeSS codebase under the 
  [BSD license](https://github.com/ElixirTeSS/TeSS/blob/master/LICENSE).

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -345,6 +345,19 @@
...
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.string "scrape_or_process"
+    t.string "model"
+    t.string "prompt"
+    t.string "input"
+    t.string "output"
+    t.boolean "needs_processing", default: false
+    t.index ["event_id"], name: "index_llm_objects_on_event_id"
+  end
```
Comment (fbacall) [inline db/schema.rb:359]: Should not be here

File: db/schema.rb [hunk 2]
```diff
@@ -451,6 +465,15 @@
...
 
+  create_table "sessions", id: :serial, force: :cascade do |t|
+    t.string "session_id", null: false
+    t.text "data"
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at", precision: nil
+    t.index ["session_id"], name: "index_sessions_on_session_id", unique: true
+    t.index ["updated_at"], name: "index_sessions_on_updated_at"
+  end
+
```
Comment (fbacall) [inline db/schema.rb:476]: Also should not be here

**Review by fbacall — COMMENTED** [review]
Looks good, just remove the extra tables that got slipped into `schema.rb`

**Review by fbacall — APPROVED** [review]

File: db/schema.rb [hunk 3]
```diff
@@ -378,6 +378,7 @@
     t.text "contact"
     t.text "learning_objectives"
     t.string "fields", default: [], array: true
+    t.boolean "visible", defaut: true
```
Comment (cwant) [inline db/schema.rb:381]: Just wanted to point out that "default" is misspelled here.

Comment (fbacall) [reply db/schema.rb:381]: Good catch. I'm slightly surprised Rails didn't complain about this when loading the schema in the CI

**Pull request merged**